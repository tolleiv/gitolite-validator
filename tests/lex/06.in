repo    libraries/external/external_module
         #-   master = @user
         -   master = @user
         -   TYPO3 = @user
         -   refs/tags/TYPO3 = @user
         C       =   @devs
         RW+D    =   CREATOR