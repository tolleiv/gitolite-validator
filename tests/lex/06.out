{t_string[1] repo}
{t_string[1] libraries/external/external_module}
{t_eol[1] }
{t_comment[2] -   master = @user}
{t_eol[2] }
{t_dash[3] }
{t_string[3] master}
{t_assign[3] }
{t_group[3] user}
{t_eol[3] }
{t_dash[4] }
{t_string[4] TYPO3}
{t_assign[4] }
{t_group[4] user}
{t_eol[4] }
{t_dash[5] }
{t_string[5] refs/tags/TYPO3}
{t_assign[5] }
{t_group[5] user}
{t_eol[5] }
{t_string[6] C}
{t_assign[6] }
{t_group[6] devs}
{t_eol[6] }
{t_string[7] RW+D}
{t_assign[7] }
{t_string[7] CREATOR}
